### PR TITLE
Db ios

### PR DESCRIPTION
**Pull request by tzufitltaboola**

### CODE REVIEW

File: lib/ios/Bluetrace/BluetraceManager.swift [hunk 1]
```diff
@@ -1,132 +0,0 @@
...
-        } else {
-            // todo: consider iOS 13.0, which has different behavior from 13.1 onwards
-            return CBPeripheralManager.authorizationStatus() == .authorized
-        }
-    }
-
-    func isBluetoothOn() -> Bool {
-        switch centralController.getState() {
-        case .poweredOff:
-            print("Bluetooth is off")
```
Comment (orenktaboola) [inline lib/ios/Bluetrace/BluetraceManager.swift:96]: if BT is off we doesn't present any error to the user. i think it should also call presentBluetoothAlert

**Comment by ButtersHub** [comment]
@tzufitltaboola @gran33 currently the method _getAllDevices_ in BleModule is not connected to the _getAllDevices_ method in DBClient, right?

File: lib/ios/Bluetrace/BluetraceManager.swift (hunk 1 above)
Comment (tzufitltaboola) [reply lib/ios/Bluetrace/BluetraceManager.swift:96]: I removed this class. This was a copy of the Singapore code and we decided not to use it.

**Comment by tzufitltaboola** [comment]
@ButtersHub the _getAllDevices_ is not connected yet, I only created the functions.

**Pull request merged**